prost-build/Cargo.toml

```diff
@@ -15,7 +15,7 @@ edition = "2021"
 rust-version = "1.56"
 
 [features]
-default = []
+default = ["pulldown-cmark", "pulldown-cmark-to-cmark"]
```

Comment:
Can you explain more why this should be enabled by default? I originally wanted this to be an opt-in feature so that users are not forced to pull this crate in.

Reply:
oh because I need to use `pulldown-cmark` and `pulldown-cmark-to-cmark` in `get_lines()` function with `default` feature. This means both `default` and `cleanup-markdown` features depend on `pulldown-cmark` and `pulldown-cmark-to-cmark` crates now. Is that ok?

Reply:
Would it make sense to just include this under `cleanup-markdown`?

Reply:
You mean to just include `pulldown-cmark` and `pulldown-cmark-to-cmark` under `cleanup-markdown`?

Reply:
Right, leave the futures as is and add this clean up as part of `cleanup-markdown`?

Reply:
This _incorrectly escape square brackets in code blocks_ bug only appears in `default` feature.
So if you want me to add this fix as part of `cleanup-markdown` feature then there's no need for this PR, because `cleanup-markdown` has already worked correctly.

Reply:
I am bit confused if that feature fixes it then why is there more code here. Maybe I am missing something and didn't look hard enough. 

I could be convinced to either 1) make this feature default 2) implement the specific parts of this feature internally to prost without the xtra dependency.